templates/email/plain_messages/berth_invoice_en.txt

```diff
@@ -25,6 +20,8 @@ Additional services:
 {% endfor %}
 {% endif %}
 
+Net price: {{ order.total_pretax_price }}€
+VAT: {{ order.total_tax_value }}&euro; ({{ order.tax_percentage }}%)
```

Comment:
Extra parentheses in the end? Same for the others as well.

Reply:
Yes, it would show like:
```
VAT: 250.00€ (24.00%)
```

Reply:
(See the just-attached screenshot)